data/changelog.txt

```diff
@@ -147,438 +150,97 @@ Add reminder effects for some medicine.
-Add ammo pouches
+Add "calories" field to it_comest.
+Terrain connections for groups other than WALL.
+Hands free mechanics.
 Update Evac Center.
-Neoprene expansion
-Add more metal constructions
-Rivtech magazine update.
-Added magazine to Military Recruit background
-Makes electrohack battery-powered
-Mark spare magazine as obsolete
-Make bio_ups an active bionic
-Implement MAG_DESTROY and MAG_EJECT
-Add recipes for some armor items
-Add missing cotton material to neoprene armsleeves.
-Add missing magazines to professions
-Add more magazine spawns
-Adds Kel-Tech SUB-2000
-Adds Saiga-12 magazines.
-Additional Sandwich Types
-New vehicle: Automatic Tractor
-Cannabis enhancement.
+Implement MAG_DESTROY and MAG_EJECT.
 Implements integral_volume for gun mods.
-More magazine pouches.
-Add proper magazine to starting gear of police detective
-Add more magazines (5.7mm and 4.6mm)
-NPC guard/follow update
-Add ammo itemgroups
-Veterinarian Clinic
-Implement disintegrating ammo linkages
-Normalize 762 ammo and add AKM magazines
-Normalize shot ammo
-Add alternative black gunpowder recipe
-Move naginata from Medieval Stuff mod
-Stillsuit
-More magazines and ammo
-New mapgen and gunstores
-Move jian sword from Medieval Stuff mod
-Adds deconstruct recipe for copper pot.
-Crafted holy symbol.
-Move qiang spear from Medieval Stuff mod
-Normalize and balance .38
-Update 762R ammo and items
-Add more food recipes
-Move baldric from Medieval Stuff mod
-Simplify coal and forging recipes
-NPC trade update
-Union Suit Alterations
-Move gambeson from Medieval Stuff mod
-Simplify coal and forging recipes, part 2.
-Ammo bandoliers
-New JSON gun basements
-Creation of 5 new overmap special campsites
-Anti-robot Emp gun
-Move tunic from Medieval Stuff mod
-3 new roadside rest stops
-Adds ammo bandoliers
-Real French Fries
-Add mechanical winch for brick walls
-Vet Clinic Update
-Buckets
-honey sandwich
-Comestibles stuff
-Acetylene tanks
-opaque hatches
-Zombie Brutes and water biter
-Vitamins part 1
-Extended maple products, crafting, farming, items and related concepts
-Move campfire hot dogs from more survival tools mod to main game
-Magazine adaptors
-Vibrator Deconstruction Recipe
-Allow the use of duct tape in place of medical tape
-Extended maple products part 2
-Doom Items
-Add magazine coloring, improve ammo/gun coloring
-Add missing whiskey recipe
-Prison Break Scenario
-Restores gummy vitamins
-Add ironshod quarter staff to Niten Ichi-Ryu
-Leather Belts hold slings, slingshots, and wrist-slingshots.
-Linolium floors in Vet Clinic
-Removed blowgun darts
-Remove dart recipe
-Deconstruct joints
-Add details to message displayed when loading world
-Display RDA for vitamins
-Have illegal drugs spawn in plastic bags
-Avoid moving objects that are examined later
-Revised 7.62x25 item description
-Scorched children
-Funeral home
-Razorclaws and Shipwreck
-Fungii
-Add alternative triffid groves
-Add 2x2 cemetery
-Reinvent the wheel
-Cable charger bionic
```

Comment:
Did this get added somewhere else?

Reply:
Nope missed it, it has a description now.